lv_conf.h

```diff
@@ -644,7 +644,10 @@
 #define LV_USE_BARCODE 1
 
 /*FreeType library*/
+#ifndef LV_USE_FREETYPE
```

Comment:
It makes difficult to update `lv_conf.h`. Anyway, it's a general problem that we shouldn't solve just for FREETYPE so I can merge this PR.